python/dataproc_templates/util/template_constants.py

```diff
@@ -270,3 +270,16 @@
 # Hive DDL Extractor Util
 HIVE_DDL_EXTRACTOR_INPUT_DATABASE = "hive.ddl.extractor.input.database"
 HIVE_DDL_EXTRACTOR_OUTPUT_GCS_PATH = "hive.ddl.extractor.output.path"
+
+#Kafka To Bq
+KAFKA_BQ_CHECKPOINT_LOCATION='kafka.to.bq.checkpoint.location'
+KAFKA_BOOTSTRAP_SERVERS='kafka.to.bootstrap.servers'
```

Comment:
Please update: kafka.to.bootstrap.servers to kafka.to.bq.bootstrap.servers

Reply:
Done